server/build.gradle

```diff
@@ -90,6 +93,7 @@ dependencies {
 
     testImplementation('org.smali:dexlib2:2.5.2')
     testImplementation project(':client')
+    testImplementation('org.wiremock:wiremock-standalone:3.10.0')
```

Comment:
Mock web server to serve files from local filesystem for checksum verification tests.